.github/workflows/cmo-make-targets.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
         if [ "${{ steps.create-pr.outputs.pull-request-url }}" == "" ]; then
           echo '::set-output name=message::No changes detected.'
         else
-          echo '::set-output name=message::PR ${{ steps.create-pr.outputs.pull-request-url }} has been ${{ steps.create-pr.outputs.pull-request-operation }}.'
+          echo '::set-output name=message::PR ${{ steps.create-pr.outputs.pull-request-url }} has been ${{ steps.create-pr.outputs.pull-request-operation || 'updated' }}.'
```

Comment:
doesn't the `echo '... || 'up...'  mess with the quoting? shouldn't it be escaped?

Reply:
Ahh, yes. Thank you.